### ISSUE TITLE
Testing Simulate.click

### ISSUE THREAD

**Issue by danielpoulson** [kentcdodds/ama#403]
Hi Kent
Refer app.login.todo
I am currently watching your React Testing workshop on FEM and I don't quite understand why passing in { button: 0 } allows the simulation of a react router transition from one link to another. Is there any info on this process that I could read.
Daniel

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
